### ISSUE TITLE
This code does not use nms during testing??????????????

### ISSUE THREAD

**Issue by hdjsjyl** [Goingqs/PyramidBox#29]
I do not find the nms part during testing. Anyone knows this reason? Any advice will be appreciated, thanks.

**Comment by Goingqs** [comment]
You can find it in detection.py

**Comment by hdjsjyl** [comment]
@Goingqs  thanks, in box_utils.py, log_sum_exp function : 
x_max = x.data.max()
return torch.log(torch.sum(torch.exp(x-x_max), 1, keepdim=True)) + x_max

this sentence introduces inf loss. Do you have some ideas?
